experiment/theory.md

```diff
@@ -1 +1,4 @@
-### Link your theory in here
\ No newline at end of file
+ Place the shear test attachment on the lower table, this attachment consists of cutter. The specimen is inserted in shear test attachment & lift the lower table so that the zero is adjusted, then apply the load such that the specimen breaks in two or three pieces.
```

Comment:
This whole thing is garbage. When you talk to the SME, please ask them to provide a proper paragraph for theory.